components/filemanager/data-binding/flat-data.md

```diff
@@ -42,7 +42,8 @@ You must also provide the correct value for the `HasDirectories` field - for the
 @code {
     private List<FlatFileEntry> FileManagerData = new List<FlatFileEntry>();
 
-    private string DirectoryPath { get; set; } = string.Empty;
+    private string RootPath { get; set; } = "root-folder-path";
+    private string DirectoryPath { get; set; } = "directory-folder-path";
```

Comment:
Is there a reason for the `DirectoryPath` to be different from the `RootPath`? As I see, there are no files in this directory, which means the `FileManager` will be initially empty. See: https://blazorrepl.telerik.com/mSkPEXOU23Yy45qA57

It is probably a good idea to set its initial value to match the `RootPath`. Thus initially the FileManager will show the root directory. 

This suggestion is applicable to all examples.

```suggestion
    private string RootPath { get; set; } = "root-folder-path";
    private string DirectoryPath { get; set; } = "root-folder-path"; // The initially opened directory will be the root directory.
```